core/domain/subtopic_validators.py

```diff
@@ -0,0 +1,159 @@
+# coding: utf-8
```

Comment:
Can you please assign this file to @aks681 in the `New structure` project inside CODOWNER files? 
`core/domain/subtopic_*.py @aks681` [line 398]

/cc @sajalasati

Reply:
Yes assigned

Reply:
(Sorry I had mentioned incorrect number line before*)